rules_go_gapic/go_gapic.bzl

```diff
@@ -110,8 +111,13 @@ def go_gapic_library(
     for path in samples:
         file_args[path] = "sample"
 
+  t = transport[0]
+  if len(transport) > 1:
```

Comment:
do you need this line and the line above? can we simply have `"+".join(transport)` work for 0- and 1-element lists?

Reply:
I'm not a python pro, and when I was testing in an online interpreter, a single element list resulted in the single string being split by character and joined by `+` haha so `["rest"]` became `"r+e+s+t"` oddly enough.

Reply:
I think it depends on whether `transport` is a list of strings or just a string (a sequence of chars). The following give different results:
```
transport="rest"
print("+".join(transport))
print("+".join([transport]))
```
=>
```
❯ python testsplit.py
r+e+s+t
rest
```